programs/sponsors/sponsors.md

```diff
@@ -0,0 +1,64 @@
+---
+section: docs
+title: Sponsors
+---
+JSON Schema uses [OpenCollective](https://opencollective.com/json-schema) and [GitHub Sponsors](https://github.com/sponsors) to gather money. This money is used to cover project expenses in a transparent way ([See the OpenCollective documentation](https://opencollective.com/how-it-works)), and your donation will help to sustain and grow the Project.
+
+While donations can be done to both OpenCollective and GitHub Sponsors, the JSON Schema funds are fully managed in OpenCollective, therefore the donations received in GitHub Sponsors will be transferred to OpenCollective.
```

Comment:
Phrasing trivia: should this read

`While donations can be made to both OpenCollective and GitHub Sponsors,`

Reply:
Thanks Matthew! Much better that way.